New version: DoubleFloats v1.3.7

JuliaRegistrator opened the pull request:
- Registering package: DoubleFloats
- Repository: https://github.com/JuliaMath/DoubleFloats.jl
- Created by: @JeffreySarnoff
- Version: v1.3.7
- Commit: 8b58185d1bc1a1c9261feb935e04ef3405f2fc3b
- Reviewed by: @JeffreySarnoff
- Reference: https://github.com/JuliaMath/DoubleFloats.jl/commit/8b58185d1bc1a1c9261feb935e04ef3405f2fc3b#commitcomment-140904758
- Description: math with more good bits
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd65795d0b05c2b4b39f0e0c99a97dedef7aef53e547378ba3a32c9835990aa19133187823bdeb701b84aa1ef9bba8a7e0693a3e52f42ebb4df3d9b1c63cc17d9030d6ffb36ecee3f4415312bec024136533d8bd237e39bfffa6f21e642f8205d4ed825580da3005cf1291e8609e7e16a34dbf69b3eb3b5345209d28c649c5428adcdf6b70de514e38a531c51ef793cffa7a32bbd9c5fc2ed867d4769b277de690e3cbc08d4e0ad4a8926da3068507a4235 -->

JeffreySarnoff commented:
[merge approved]

JeffreySarnoff commented:
@admin this did not skip a patch number -- this update was entered before the prior had completed.  The same happened with the following version. Thanks.

GunnarFarneback commented:
[noblock]
Automerge always compares version numbers to what is in the General repository at the moment, not to what may or may not be added by other pull requests. You can force automerge to run again by making a new registration of the same commit, but it won't help getting past:
> * I was not able to load the package (i.e. import DoubleFloats failed). See the AutoMerge logs for details.

JuliaRegistrator committed:
New version: DoubleFloats v1.3.7

UUID: 497a8b3b-efae-58df-a0af-a86822472b78
Repo: https://github.com/JuliaMath/DoubleFloats.jl.git
Tree: c56bc42569da08078146ec03d3c367e52b90db26

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

JeffreySarnoff commented:
[noblock]
@admin please allow this -- there are clients waiting for both this and the following (as yet unregistered) one

GunnarFarneback commented:
I have no powers to merge PRs but it would have gone in automatically after you fixed the issues if you had edited all your comments with `[noblock]`.
